src/common/commandExecutor.ts

```diff
@@ -135,7 +195,7 @@ export class CommandExecutor {
         });
 
         result.stdout.on("data", (data: Buffer) => {
-           Log.logStreamData(data, process.stdout);
+            Log.logStreamData(data, process.stdout);
```

Comment:
I think that this is actually fixing the spacing issue; it is indented 3 steps, and has 12 spaces so I think it's correct now.